src/main/java/Board.java

```diff
@@ -1,6 +1,40 @@
 import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
 
 public class Board {
-    Hex[] board = new Hex[16];
+    //Hex[] board = new Hex[16];
+    List<Hex> board = new ArrayList<>();
+
+    public Board(){
+        for (int i = 0 ; i < 3; i++){
+            board.add( new Hex(ResourceType.ORE));
+        }
+
+        for (int i = 4 ; i < 7; i++){
```

Comment:
Do you need to do it that way? you can probably start at 0 for each

Reply:
You're right. Did it that way because Nina had put in position, but i removed position in my pr